examples/lerax-square.cpp

```diff
@@ -53,75 +67,35 @@ struct Player {
     }
 };
 
-class Game {
+
+class Game: public GameLoop {
 private:
-    SDL_Window *window = nullptr;
-    SDL_Renderer *renderer = nullptr;
-    const char *title = "Square Moving";
-    bool running = true;
-    struct Player player;
+    using GameLoop::GameLoop;
```

Comment:
I would rather prefer you to place this expression inside a function where it's used. It's quite cumbersome to read this inside a class declaration let alone interpret this in the scope of the class.

Reply:
I was trying to kill a annoying compiling error. I can move that to `main` if that still fixes it, but I'm not sure. BTW, why the constructor is not inherited by default?

Reply:
I already have suggest to clean the syntax in future versions, instead of inherit a class, only declare the common functions like: draw(), update(), keyevent() and etc.Limiting unnecessary back-end information to the developer and make syntax more clean and fast to code.

Reply:
I already have suggest to clean the syntax in future versions, instead of inherit a class, only declare the common functions like: draw(), update(), keyevent() and etc.Limiting unnecessary back-end information to the developer and make syntax more clean and fast to code.

Reply:
That can be made including a new header called `Game.hpp` containing only declarations

Reply:
We should create a issue only to get rid of that particular problem of `GameLoop::GameLoop`